src/views/ingresses/details/components/IngressDetailsPageTitle/IngressDetailsPageTitle.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+import { IngressModel, modelToRef } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Flex,
+  FlexItem,
+  PageSection,
+  PageSectionVariants,
+  Stack,
+  Title,
+} from '@patternfly/react-core';
+import { useLastNamespacePath } from '@utils/hooks/useLastNamespacePath';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { getName } from '@utils/resources/shared';
+import IngressActions from '@views/ingresses/actions/IngressActions';
+
+import './IngressDetailsPageTitle.scss';
+
+type IngressDetailsPageTitleProps = {
+  ingress: IoK8sApiNetworkingV1Ingress;
+};
+
+const IngressDetailsPageTitle: FC<IngressDetailsPageTitleProps> = ({ ingress }) => {
+  const { t } = useNetworkingTranslation();
+  const namespacePath = useLastNamespacePath();
+
+  return (
+    <PageSection className="co-m-nav-title--detail" variant={PageSectionVariants.light}>
```

Comment:
Does this cn exist co-m-nav-title--detail?

Reply:
Yes. If you remove it the border between the title and the tabs disappears.